elastic/kibana
# [Files] Fix collapsing UI

## Summary

* Added the ability to pass a class name to the upload component container
* Set a min-width for the uploader component that looks better from within FilePicker
* Added a delay to the storybook so that the loading state can be tested on the `Empty` version of the file picker story

## How to test

1. `yarn storybook files`
2. Check out the `Empty` view of the file picker
3. Try to add a file with a looooooong name and try to add a file with a short name

Component should look good in both cases

## Screenshots

### Before

<img width="269" alt="Screenshot 2022-11-17 at 15 39 12" src="https://user-images.githubusercontent.com/8155004/202475783-2ad86d15-61f5-4c4a-8400-fcdda0eec07b.png">

### After

<img width="537" alt="Screenshot 2022-11-17 at 15 40 38" src="https://user-images.githubusercontent.com/8155004/202475987-8d9de52d-f3ff-4053-a08e-229b43952513.png">

<img width="531" alt="Screenshot 2022-11-17 at 15 57 16" src="https://user-images.githubusercontent.com/8155004/202480284-9212d59a-a27d-4e5a-a488-41d5f9af8538.png">
